### PR TITLE
ci: switch to github actions

### PR DESCRIPTION
**Pull request by fjmolinas**
This PR removes travis-ci and docker-hub hooks, bors should now check the the github action build result when pushing to `trying` and `staging`...

Testing procedure:
- run `bors try` and it should look at the github action result.

### CODE REVIEW

**Comment by fjmolinas** [comment]
bors try

**Comment by fjmolinas** [comment]
Looks like the status checks is wrong..

File: bors.toml [hunk 1]
```diff
@@ -1 +1 @@
-status = ["continuous-integration/travis-ci/push"]
+status = ["build"]
```
Comment (kaspar030) [inline bors.toml:1]: I think this must be `build / Build and Test (push)`.

I don't know where to find this easily. I clicked on the "code" tab of the repo, changed the branch to trying. There is a green check sign next to the commit. If I click that, I see "build / Build ..." abbrevated. I used firefox' "Inspect element" to see the full text...

**Comment by fjmolinas** [comment]
bors cancel

**Comment by fjmolinas** [comment]
bors try

**Comment by kaspar030** [comment]
bors try

File: bors.toml (hunk 1 above)
Comment (kaspar030) [reply bors.toml]: did you try this?

**Comment by fjmolinas** [comment]
Seems that it didn't work either, trying with your suggestion

File: bors.toml [hunk 2]
```diff
@@ -1 +1 @@
-status = ["continuous-integration/travis-ci/push"]
+status = ["build / Build and Test (push)"]
```
Comment (miri64) [inline bors.toml:1]: Just guessing, but maybe just

```suggestion
status = ["build / Build and Test"]
```

The thing in parens is just the trigger (see in the PR checks where it is `(pull request)`.

Comment (miri64) [reply bors.toml:1]: If https://www.christopherbiscardi.com/integrating-bors-with-github-actions is to believe, I think it should be

```suggestion
status = ["Build and Test"]
```

**Comment by kaspar030** [comment]
bors try

File: bors.toml (hunk 2 above)
Comment (fjmolinas) [reply bors.toml]: Lets see

**Comment by miri64** [comment]
bors try

**Comment by miri64** [comment]
bors help

**Comment by miri64** [comment]
Can one tell boars to stop trying? The action [already succeeeded](https://github.com/RIOT-OS/riotdocker/actions/runs/914473347).

**Comment by fjmolinas** [comment]
> Can one tell boars to stop trying? The action [already succeeeded](https://github.com/RIOT-OS/riotdocker/actions/runs/914473347).

I tried with `bors cancel` but it doesn't seem to work...

**Comment by kaspar030** [comment]
bors cancel

**Comment by kaspar030** [comment]
bors ping

**Comment by kaspar030** [comment]
bors help

**Comment by kaspar030** [comment]
bors try-

**Comment by kaspar030** [comment]
bors try

**Comment by miri64** [comment]
> bors try-

Was that what did the trick?

**Comment by kaspar030** [comment]
yup, "bors cancel" cancels a merge request, "bors try-" the trying... https://bors.tech/documentation/

**Comment by fjmolinas** [comment]
> yup, "bors cancel" cancels a merge request, "bors try-" the trying... https://bors.tech/documentation/ -> "Reference"

Who would have though we only needed to read the doc hahahah

**Comment by kaspar030** [comment]
> Who would have though we only needed to read the doc hahahah

well you'd expect a UI to work intuitively for smart people like us. :sunglasses:

File: bors.toml (hunk 2 above)
Comment (fjmolinas) [reply bors.toml]: That was it

**Comment by fjmolinas** [comment]
> ## try
> 
> Build succeeded:
> 
>     * [Build and Test](https://github.com/RIOT-OS/riotdocker/runs/2763682909)

Should I squash?

**Comment by kaspar030** [comment]
yes please!

**Comment by miri64** [comment]
> yup, "bors cancel" cancels a merge request, "bors try-" the trying... https://bors.tech/documentation/ -> "Reference"

Maybe it's just me, but I was searching for such a cheat sheet and could not find it... maybe need to reevaluate my googling skills ^^"

File: bors.toml (hunk 2 above)
Comment (kaspar030) [reply bors.toml:1]: is this back to before or is github's ui tricking me?

**Review by miri64 — APPROVED** [review]
bors merge

**Comment by kaspar030** [comment]
bors cancel

**Comment by fjmolinas** [comment]
Wait I forgot to pull the changes.

**Comment by kaspar030** [comment]
rebase mishap I think

**Comment by kaspar030** [comment]
Now please @miri64 :)

**Review by miri64 — APPROVED** [review]
bors merge

**Comment by miri64** [comment]
(testing btw if it also catches review comments ;-))

**Comment by kaspar030** [comment]
> (testing btw if it also catches review comments ;-))

seems so!

Next thing would be to configure bors to require rewiews.

**Comment by miri64** [comment]
> > (testing btw if it also catches review comments ;-))
> 
> seems so!
> 
> Next thing would be to configure bors to require rewiews.

unlimited power GIF

**Comment by miri64** [comment]
> > > (testing btw if it also catches review comments ;-))
> > 
> > 
> > seems so!
> > Next thing would be to configure bors to require rewiews.
> 
> unlimited power GIF

:-/ remember the time when gifs were easy :roll_eyes: ...

**Pull request merged**